msal_extensions/windows.py

```diff
@@ -39,6 +39,15 @@ def raw(self):
         _MEMCPY(blob_buffer, pb_data, cb_data)
         return blob_buffer.raw
 
+_err_description = {
+    # Keys came from real world observation, values came from winerror.h (https://errors)
+    -2146893813: "Key not valid for use in specified state.",
+    -2146892987: "The requested operation cannot be completed. "
+        "The computer must be trusted for delegation and "
+        "the current user account must be configured to allow delegation. "
+        "See also https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/enable-computer-and-user-accounts-to-be-trusted-for-delegation",
```

Comment:
Great!

Reply:
I don't think people can understand this enigma.

Reply:
> I don't think common users can understand this enigma. They are still non-actionable.

That is arguably true. (TBH, that link was a TL;DR for me. LOL.) But my thoughts were:
* Bogdan found this valuable information that might be useful to those who are determined enough to follow that route, so I would keep it here rather than waste it.
* These paragraphs are for the "inner exception" `OSError`. The "outer exception" `PersistenceEncryptionError` and `PersistenceDecryptionError` both end with a slightly-more-actionable sentence.
* Eventually, Azure CLI may still want to catch the `PersistenceEncryptionError` and `PersistenceDecryptionError`, and convert them into an end-user-friendly sentence.